ResidentsSocialCarePlatformApi.Tests/V1/Gateways/SocialCare/GetCaseNotesTests.cs

```diff
@@ -0,0 +1,90 @@
+using System.Linq;
+using FluentAssertions;
+using NUnit.Framework;
+using ResidentsSocialCarePlatformApi.Tests.V1.Helper;
+using ResidentsSocialCarePlatformApi.V1.Domain;
+using ResidentsSocialCarePlatformApi.V1.Gateways;
+using ResidentsSocialCarePlatformApi.V1.Infrastructure;
+
+namespace ResidentsSocialCarePlatformApi.Tests.V1.Gateways.SocialCare
+{
+    public class GetCaseNotesTests : DatabaseTests
+    {
+        private SocialCareGateway _classUnderTest;
+
+        [SetUp]
+        public void Setup()
+        {
+            _classUnderTest = new SocialCareGateway(SocialCareContext);
+        }
+
+        [Test]
+        public void WhenThereAreNoMatchingRecords_ReturnsEmptyList()
+        {
+            var response = _classUnderTest.GetCaseNotes(123);
+
+            response.Should().BeEmpty();
+        }
+
+        [Test]
+        public void WhenThereIsOneMatch_ReturnsCaseNoteInformationForAGivenPersonId()
+        {
+            var person = AddPersonToDatabase();
+            AddCaseNoteToDatabase(person.Id);
+
+            var response = _classUnderTest.GetCaseNotes(person.Id);
+            response.FirstOrDefault().Should().NotBe(null);
```

Comment:
For readability, could we add a newline to separate the "act" and "assertion" of this test?

```suggestion
            var response = _classUnderTest.GetCaseNotes(person.Id);
            
            response.FirstOrDefault().Should().NotBe(null);
```

Reply:
👍🏾